static_files/genesis-generation-config/cl/config.yaml.tmpl

```diff
@@ -34,7 +34,7 @@ TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH: 18446744073709551615
 
 # Capella
 CAPELLA_FORK_VERSION: 0x40000038
-CAPELLA_FORK_EPOCH: {{ .CapellaForkEpoch }}
+CAPELLA_FORK_EPOCH: 0
```

Comment:
if we leave this as something that is passed then we can fail loudly if MEV is run against capella 0 or with mev pass a >0 capella